src/strings/en-us.json

```diff
@@ -599,6 +599,8 @@
     "LabelBaseUrlHelp": "Add a custom subdirectory to the server URL. For example: <code>http://example.com/<b>&lt;baseurl&gt;</b></code>",
     "LabelBackdropScreensaverInterval": "Backdrop Screensaver Interval",
     "LabelBackdropScreensaverIntervalHelp": "The time in seconds between different backdrops when using the backdrop screensaver.",
+    "LabelScreensaverTimeHelp": "The amount of time in seconds of inactivity required to start the screensaver.",
+    "LabelScreensaverTime": "Screensaver Time",
```

Comment:
Add new strings in alphabetical order (move below and swap).

Reply:
Changes applied, thank you

Reply:
Also move them below. They should be after: https://github.com/jellyfin/jellyfin-web/blob/b7995eee4a9b13bb67d4adce03da726f22621f6f/src/strings/en-us.json#L865

Reply:
Fixed that as well